### PR TITLE
[project-base] fixed console warnings and errors

### PR DESCRIPTION
**Pull request by chlebektomas**
#### Description, the reason for the PR
Fixed console warnings and errors for better UX and DX.

#### Fixes issues
... 

#### Check the appropriate checkboxes below, please

- [ ] [BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/) 
- [ ] New feature 
- [x] I have read and signed [License Agreement for contributions](https://www.shopsys.com/license-agreement)

----
:globe_with_meridians: Live Preview:
  - https://tc-ssp-2680-fixed-console-warnings-and-errors.odin.shopsys.cloud
  - https://cz.tc-ssp-2680-fixed-console-warnings-and-errors.odin.shopsys.cloud

### CODE REVIEW

**Review by JanMolcik — APPROVED** [review]

File: project-base/storefront/components/Pages/Customer/EditProfile/BillingAddress.tsx [hunk 1]
```diff
@@ -18,21 +17,8 @@ export const BillingAddress: FC = () => {
...
-        if (countriesAsSelectOptions.length > 0) {
-            const userCountryValue = formProviderMethods.getValues(formMeta.fields.country.name);
-            const selectedCountry =
-                countriesAsSelectOptions.find((option) => option.value === userCountryValue.value) ||
-                countriesAsSelectOptions[0];
-
-            setValue(formMeta.fields.country.name, selectedCountry, { shouldValidate: true });
-        }
-    }, [countriesAsSelectOptions, formMeta.fields.country.name, formProviderMethods, setValue]);
-
```
Comment (JanMolcik) [inline project-base/storefront/components/Pages/Customer/EditProfile/BillingAddress.tsx]: This change will remove pre-selecting of users country, is that intentional?

**Review by JanMolcik — CHANGES_REQUESTED** [review]

File: project-base/storefront/components/Pages/Customer/EditProfile/BillingAddress.tsx (hunk 1 above)
Comment (chlebektomas) [reply project-base/storefront/components/Pages/Customer/EditProfile/BillingAddress.tsx]: Users must add their Billing address during registration, so there is no need for a pre-selecting option from the select box.

Comment (chlebektomas) [reply project-base/storefront/components/Pages/Customer/EditProfile/BillingAddress.tsx]: Actually, users who log in via social networks don't have any info. So I fixed it.

File: project-base/storefront/tailwind.config.js [hunk 2]
```diff
@@ -92,19 +92,6 @@ module.exports = {
...
-            secondaryTextDisabled: '#B6C3D8',
-            secondaryTextHovered: '#004EB6',
-            secondaryBackground: '#004EB6',
-            secondaryBackgroundActive: '#004EB6',
-            secondaryBackgroundDisabled: '#FAFAFA',
-            secondaryBackgroundHovered: '#FFFFFF',
-            secondaryBorder: '#004EB6',
-            secondaryBorderActive: '#004EB6',
-            secondaryBorderDisabled: '#B6C3D8',
-            secondaryBorderHovered: '#004EB6',
```
Comment (JanMolcik) [inline project-base/storefront/tailwind.config.js:106]: Don't we want to keep it here even though we don't use them as a lead for future projects how to use the colour system?

**Review by JanMolcik — CHANGES_REQUESTED** [review]

**Review by JanMolcik — APPROVED** [review]

**Review by JanMolcik — APPROVED** [review]

**Pull request merged**